chain-signatures/node/src/storage/presignature_storage.rs

```diff
@@ -96,10 +110,17 @@ impl PresignatureRedisStorage {
         Ok(result)
     }
 
+    pub async fn len_used(&self) -> PresigResult<usize> {
```

Comment:
seems like we never clear the used presignatures. So the used presignatures will grow indefinitely as we generate and use more presignatures. 
I think we should add a mechanism that get rid of the very old presignatures in the used_key

Reply:
Redis allows to add time it will be stored in the DB, we can use that